### DISCUSSION TITLE
Polymorphism

### DISCUSSION THREAD

**Discussion by Eugene-development** [prisma/prisma#16381] [Q&A] [answered]
How can i implement polymorphic table linking? For example
https://sequelize.org/docs/v6/advanced-association-concepts/polymorphic-associations/

**Comment by nurul3101** [comment, marked as answer]
Hello @Eugene-development 👋 

Polymorphism is not natively supported by Prisma yet. We have a feature request for it here: #1644, if you can leave a 👍 to the feature request then it would help us in prioritising it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
